### PR TITLE
script for finder application added

### PR DESCRIPTION
**Pull request by christianhultin**

### CODE REVIEW

File: apps/finder.py [hunk 1]
```diff
@@ -0,0 +1,34 @@
+from talon.voice import Key, Context, Str, press
+
+def go_to_path(path):
+	def path_function(m):
```
Comment (dwiel) [inline apps/finder.py]: could you please convert tabs into spaces

File: apps/vscode.py [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+    # Finding text
+    'find': Key('cmd-f'),
+    'find next <dgndictation>': jump_to_next_word_instance,
+
+    # Clipboard
+    'clone': Key('alt-shift-down'),
+    
+    # Navigation
+    'line' + threeDigitNumber: jump_to_line,
+    'Go to line': Key('cmd-g'),
```
Comment (dwiel) [inline apps/vscode.py]: if you make the three digit number optional, you can combine this into a single command. Check out `utils.optional_numerals`

File: apps/vscode.py [hunk 3]
```diff
@@ -0,0 +1,87 @@
+from talon.voice import Context, Key, press, Str
+from user.utils import parse_words_as_integer
```
Comment (dwiel) [inline apps/vscode.py]: these should be relative, so that other people can put this code in some other path if they want to. `from ..utils import`

Comment (dwiel) [reply apps/vscode.py]: also this line isn't necessary since parse_words_as_integer is already imported below

Comment (christianhultin) [reply apps/vscode.py]: Done

File: apps/vscode.py (hunk 2 above)
Comment (christianhultin) [reply apps/vscode.py]: sorry completely forgot that I had this in another file where it was optional already, but I used yours instead. Don't know what you mean by one command though

File: apps/finder.py (hunk 1 above)
Comment (christianhultin) [reply apps/finder.py]: It is weird, seems like pylint is not working on my machine and that makes it impossible to change to spaces, it changes back to tabs every time I close the file :/

Comment (christianhultin) [reply apps/finder.py]: got it...

**Pull request merged**